plugins/ci/pkg/ci/auto_detection_test.go

```diff
@@ -39,6 +39,9 @@ func TestAutoDetection(t *testing.T) {
 				},
 			},
 		},
+		Terraform: models.TerraformConfig{
```

Comment:
You may want to copy a valid tf file on `./testdata/repo1` to test the auto-detection